lib/memento/mnesia.ex

```diff
@@ -74,6 +64,10 @@ defmodule Memento.Mnesia do
       {:error, reason} ->
         {:error, reason}
 
+      {:aborted, reason = {:no_majority, _}} ->
+        call(:report_event, [{:minority_write_attempt, node()}])
+        {:error, reason}
```

Comment:
This is the "hook" in Memento "core" to make minority island write fail cases observable to the supervisor.

1. Write attempt fails.
2. This result is caught here and reported to the Mnesia system event log.
3. Supervisor is subscribe (at its init) so it "see" each of these when they happen -- and can take action.

Reply:
Sounds perfect.